code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -88,9 +88,9 @@
 /mob/living/carbon/xenomorph/proc/handle_critical_health_updates()
 	var/turf/T = loc
 	if((istype(T) && locate(/obj/effect/alien/weeds) in T))
-		heal_wounds(XENO_RESTING_HEAL + warding_aura * 0.5) //Warding pheromones provides 0.25 HP per second per step, up to 2.5 HP per tick.
+		heal_wounds(XENO_RESTING_HEAL + (warding_aura * 0.5) * 0.5) //Warding pheromones provides 0.25 HP per second per step, up to 2.5 HP per tick. //Values halved -SplinterGP
```

Comment:
then change the comment to be correct

Reply:
oh sorry heck, i swear i had put a comment like that,

Reply:
done